docs/concepts/whatisvpp/whatisvpp.rst

```diff
@@ -0,0 +1,363 @@
+.. _whatisvpp01:
+
+.. toctree::
+
+Introduction
+------------
+
+The VPP platform is an extensible framework that provides out-of-the-box
+production quality switch/router functionality. It is the open source
+version of Cisco's Vector Packet Processing (VPP) technology: a high
+performance, packet-processing stack that can run on commodity CPUs.
+
+The benefits of this implementation of VPP are its high performance,
+proven technology, its modularity and flexibility, and rich feature set.
+
+The VPP technology is based on proven technology that has helped ship
+over $1 Billion of Cisco products. It is a modular design. The framework
```

Comment:
Instead of "The VPP technology is based on proven technology that has helped ship over $1 Billion of Cisco products".

I would word this a as "FD.io VPP is a productized commercial-grade network stack that has been use in products since 2002, by a number of vendors including Cisco and ZTE".